extension/observer/k8sobserver/factory.go

```diff
@@ -25,7 +24,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
-	"k8s.io/client-go/tools/clientcmd"
+
+	k8sconfig "github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/kubernetes"
```

Comment:
What about calling the package something that does not conflict with the main package. Maybe `k8sconfig` is a better name.